common/bigquery/geonames/01_create_names.sql

```diff
@@ -0,0 +1,118 @@
+CREATE OR REPLACE TABLE `@@BQ_PREFIX@@carto.names` (
```

Comment:
We'll remove this (it's handled by DO now)

Reply:
:fire: